### ISSUE TITLE
Getting problems with first line - upon first opening terminal & ANY PowerShell.

### ISSUE THREAD

**Issue by RobFosterNYC** [microsoft/terminal#12100]
### Windows Terminal version

1.11.3471.0

### Windows build number

10.0.19044.0

### Other Software

Using "Oh-My-Posh" customized Prompt.

### Steps to reproduce

Just open Windows Terminal with customized prompt  "Oh-My-Posh"

### Expected Behavior

For icons to show on first line properly.

### Actual Behavior

When opening the prompt each time, only the first line in the prompt is affected with some sort of glitch
that has vertical lines running through the "Oh-My-Posh" customized prompt.

Please refer to this post as it gives more details, when I posted it on PowerShell Github issues page.

https://github.com/PowerShell/PowerShell/issues/16690

Here is a reference picture of the problem.

![Screenshot_90](https://user-images.githubusercontent.com/18384135/148317596-78fe1e1c-eefc-4f28-a887-e9b5ddefbedb.jpg)

**Comment by 237dmitry** [comment]
How does it look in the standard console host? `Win+R` --> `pwsh -nol` (with the same font to display the powerline correctly )
If it's the same that the problem is rather `oh-my-posh` or maybe `pwsh`

**Comment by RobFosterNYC** [comment]
You can see something breaking on the first line. Black lines going vertical just like the normal version.
.
![Screenshot_95](https://user-images.githubusercontent.com/18384135/148363844-5c484b5a-3f53-44de-96a1-4a8298c32031.jpg)

**Comment by 237dmitry** [comment]
So, this is not Windows Terminal. I would ask the [oh-my-posh](https://github.com/JanDeDobbeleer/oh-my-posh/issues) developers

**Comment by j4james** [comment]
This is a duplicate of #8341. Some users have reported that it's resolved by switching to WSL from the Microsoft Store - https://github.com/microsoft/terminal/issues/8341#issuecomment-940657032.

**Comment by RobFosterNYC** [comment]
> This is a duplicate of #8341. Some users have reported that it's resolved by switching to WSL from the Microsoft Store - [#8341 (comment)](https://github.com/microsoft/terminal/issues/8341#issuecomment-940657032).

Aah, the new WSL is only for people running windows 11, not windows 10.
I have checked my Windows update and I currently have the latest update for WSL for my version of windows.

I did try the < Clear-Host > at the end of my $PROFILE file.
And that seems to work.  Also this is not a Oh-My-Posh  issue and some may believe. This is a Terminal Issue.
I was even told by PowerShell people that its also not them as well.

Hope this gets resolved in the new update or so.
